package.json

```diff
@@ -67,7 +68,7 @@
   },
   "scripts": {
     "show-linked-packages": "find node_modules node_modules/@* -depth 1 -type l -print",
-    "link-packages": "grep -oE '@shapeshiftoss\/[a-z-]*' package.json | grep -v hdwallet | xargs yarn link",
```

Comment:
change this back please yarn keeps munging it, i need to move this to a script

Reply:
Done.